desktop/src/main/java/bisq/desktop/main/support/dispute/DisputeView.java

```diff
@@ -1139,14 +1139,14 @@ public void updateItem(final Dispute item, boolean empty) {
                                 super.updateItem(item, empty);
 
                                 if (item != null && !empty) {
-                                    field = new HyperlinkWithIcon(item.getShortTradeId());
                                     Optional<Trade> tradeOptional = tradeManager.getTradeById(item.getTradeId());
                                     if (tradeOptional.isPresent()) {
+                                        field = new HyperlinkWithIcon(item.getShortTradeId());
                                         field.setMouseTransparent(false);
                                         field.setTooltip(new Tooltip(Res.get("tooltip.openPopupForDetails")));
                                         field.setOnAction(event -> tradeDetailsWindow.show(tradeOptional.get()));
                                     } else {
-                                        field.setMouseTransparent(true);
+                                        setText(item.getShortTradeId());
                                     }
```

Comment:
This is all you need.

`Optional<Trade> tradeOptional = tradeManager.getTradeById(item.getTradeId());  `

`if ((item != null && !empty) && tradeOptional.isPresent()) {`

 `  setText(item.getShortTradeId());`

`} else {`

 `  setText("");`

`}`

Every other line related to the hyperlink can be deleted(1135, 1151 - 1155). 
Might want to use getDateColumn() above(1097) for reference. Its also a text with no hyperlink.

Reply:
The hyperlink is needed for the DisputeClientView that is extending DisputeView. So if a trade is available it should link to the trade details popup.